pages/overview/introduction.zh.mdx

```diff
@@ -0,0 +1,74 @@
+import { DimmedText } from '../../components/DimmedText';
+import { InfoPanel } from '../../components/InfoPanel';
+
+# 介绍
+
+## 什么是 Opire？
+
+Opire 是一个**面向软件开发人员的奖励平台**。
+
+这个项目诞生于<u>一个明确的目标</u>：**任何人都可以从开源软件中获得收入**。这包括那些希望专注于开发软件和改进开源代码的程序员，以及那些通过他们的项目为社区做出贡献的开源代码的所有者和维护者。
+
+有了 Opire，**任何人都可以为开源项目创建奖励**并发展他们的社区，而开发人员可以解决问题并**获得相应的奖励**。
+
+## 它是如何工作的？
+
+Opire 由两个组件组成：一个[机器人](/overview/install-bot)（在 GitHub 上安装）和一个[网络平台](https://app.opire.dev)。
+
+该机器人可以直接与 GitHub 集成，使用不同的[命令](/overview/commands)与平台进行交互：创建奖励、认领奖励等。这使得在开发人员的工作空间内能够更舒适、更一体化地进行操作。
+
+奖励也可以从网络平台上创建和认领，该平台还专注于奖励的可视化和管理。除了其他功能，它还允许进行高级搜索（按技术、价格等）、支付已认领的奖励、您解决的问题的历史记录或配置您的个人资料。
+
+详细了解奖励生命周期，请查看此[部分](/rewards/lifecycle)。
+
+## 为什么使用 Opire？
+
+<br/>
+<InfoPanel>要探索使用 Opire 的其他原因，请查看可能对您感兴趣的一些[用例](/overview/use-cases)。</InfoPanel>
+
+### 对开源维护者的好处
+
+#### 您不会失去任何东西
+<DimmedText>我们的[机器人](/overview/install-bot)是完全免费的。安装它不会让您失去任何东西（只需要几秒钟！），您可以立即从中受益。</DimmedText>
+
+#### 赚取金钱
+<DimmedText>如果项目外的某人在您的项目中创建了奖励，当奖励支付时，您将获得一定的百分比。这可以成为维护和改进您的开源项目的有价值的激励。</DimmedText>
+
+#### 提升知名度
+<DimmedText>在您的项目中创建的奖励将吸引有兴趣认领它们的开发人员。这将使您的项目成为优秀专业人士关注的焦点。</DimmedText>
+
+#### 更强大和完整的软件
+<DimmedText>通过启用为您的问题创建奖励，您的项目将快速发展和改进。这种加速的开发周期会导致更强大和更具韧性的软件，从而使您的用户和声誉受益。</DimmedText>
+
+#### 扩展您的社区
+<DimmedText>允许创建奖励不仅会加强您的社区，还会促进您的项目中的协作和参与文化。</DimmedText>
+
+#### 让您的项目保持活力（即使您没有时间）
+<DimmedText>如果由于您的情况，您无法像您希望的那样投入时间来维护您的项目，那就向其他开发人员敞开大门，让他们来帮助您。</DimmedText>
+
+### 对创建奖励的个人/组织的好处
+
+#### 使团队专注于重要的事情
+<DimmedText>
+通过请求对第三方库进行修改，您可以在不损害开发人员团队资源的情况下获得战略优势。这种灵活性使您的团队能够专注于可以增加价值的领域。
+</DimmedText>
+
+#### 继续使用您喜爱的库
+<DimmedText>
+您不必停止使用您那么喜爱的库。那个令人讨厌的 bug（没有人注意到）不会阻止您：为开发人员创建一个奖励来解决您的问题。
+</DimmedText>
+
+#### 更快地解决问题
+<DimmedText>
+通过激励开发人员解决对您重要的问题，您的项目依赖关系将更快地改善。
+</DimmedText>
+
+#### 帮助开源项目提升水平
+<DimmedText>
+您是否曾经遇到过一个看起来很有前途但不够精致的开源项目？创建奖励，让其他人改进它！
+</DimmedText>
+
+#### 支持开源
+<DimmedText>
+您是否认为世界欠数千个开源项目一个债？为您最喜欢的项目之一做出贡献：[为其中的一个问题创建奖励](/overview/commands#create-reward)或为其创建者[给予小费](/overview/commands#tip-a-user)。
```

Comment:
Relative links are still pointing to the English version. Should be something like `(/zh/overview/commands#whatever-the-id-is-in-chinese)`